app/views/admin/organisations/index.html.erb

```diff
@@ -1,33 +1,102 @@
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<% if @organisations.present? %>
+  <div class="app-view-organisation-index__table">
+    <%= render "govuk_publishing_components/components/table", {
+      filterable: true,
+      label: "Filter organisations",
+      head: [
+        {
+          text: "Acronym"
+        },
+        {
+          text: "Name",
+        },
+        {
+          text: "Type",
+        },
+        {
+          text: "Status",
+        },
+        {
+          text: "Link",
+        }
+      ],
+      rows: [
+        if @user_organisation.present?
+          [
+            {
+              text: (link_to(
+                  @user_organisation.acronym,
+                  admin_organisation_path(@user_organisation),
+                  class: "govuk-link govuk-!-font-weight-bold"
+                ) if @user_organisation.acronym.present?) || "",
+            },
+            {
+              text: link_to(
+                  @user_organisation.name,
+                  admin_organisation_path(@user_organisation),
+                  class: "govuk-link govuk-!-font-weight-bold"
+                ),
+            },
+            {
+              text: tag.p(
+                @user_organisation.organisation_type.name,
+                class: "govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-margin-top-0"
+              ),
+            },
+            {
+              text: tag.p(
+                @user_organisation.govuk_status,
+                class: "govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-margin-top-0"
+                ),
+            },
+            {
+              text: link_to('[gov.uk]', @user_organisation.public_path, class: "govuk-link govuk-!-font-weight-bold") +
+                      (link_to('[current site]', @user_organisation.url, class: "govuk-link govuk-!-font-weight-bold") if @user_organisation.govuk_status != 'live'),
+            },
+          ]
+        end
+      ].compact +
+        @organisations.map do |organisation|
+          [
+            {
+              text: (link_to(
+                  organisation.acronym,
+                  admin_organisation_path(organisation),
+                  class: "govuk-link"
+                ) if organisation.acronym.present?) || "",
+            },
+            {
+              text: link_to(
+                  organisation.name,
+                  admin_organisation_path(organisation),
+                  class: "govuk-link",
+                ),
+            },
+            {
+              text: organisation.organisation_type.name,
+            },
+            {
+              text: organisation.govuk_status,
+            },
+            {
+              text: link_to('[gov.uk]', organisation.public_path, class: "govuk-link") +
+                      (link_to('[current site]', organisation.url, class: "govuk-link") if organisation.govuk_status != 'live'),
+            }
+          ]
+        end
+    } %>
```

Comment:
should this logic be moved up to the top?

Reply:
tbh i think it should go into a helper method and be tested. There's quite a lot going on in it. I'll sort that this morning.